apps/client/src/app/pages/accounts/create-or-update-account-dialog/create-or-update-account-dialog.component.ts

```diff
@@ -47,8 +58,22 @@ export class CreateOrUpdateAccountDialog implements OnDestroy {
       currency: [this.data.account.currency, Validators.required],
       isExcluded: [this.data.account.isExcluded],
       name: [this.data.account.name, Validators.required],
-      platformId: [this.data.account.platformId]
+      platformId: [
+        this.platforms.find(
+          (platform) => platform.id === this.data.account.platformId
```

Comment:
Can you use the more explicit form for consistency?

```
({id}) => { return id === this.data.account.platformId; }
```

Reply:
thanks for the feedback, didnt know about the explicit form :D quite a long time since my last work with typescript